src/ChatBubble/index.tsx

```diff
@@ -15,6 +16,21 @@ export default class ChatBubble extends React.Component {
     super(props);
   }
 
+  renderMessageString = text => (
+    <p style={{ ...styles.p, ...text }} dangerouslySetInnerHTML={{
+      __html: anchorme(text, {
```

Comment:
I'm not sure this is the solution I would use. I'd probably use regex's! What do you think??